src/app/components/core/notifications/notification-service-config/notification-service-config.component.html

```diff
@@ -0,0 +1,5 @@
+<a *ngIf="rolesService.hasControlAccess()" class="button" (click)="openConfigureModal()" id="config-icon">
+    <span class="icon">
+        <i class="bi bi-gear-wide-connected bi-sm" aria-hidden="true" title="Notification Service Configuration"></i>
+    </span>
+</a>
```

Comment:
We already have an identical component `dispatcher-service-config.component`. Better to make that generic component instead separate for each service.